src/types/onyx/Report.ts

```diff
@@ -218,6 +218,9 @@ type Report = OnyxCommon.OnyxValueWithOfflineFeedback<
         /** For expense reports, this is the total amount requested */
         unheldTotal?: number;
 
+        /** Total amount of unheld and non-reimbursable transactions in an expense report */
```

Comment:
```suggestion
        /** Total amount of unheld non-reimbursable transactions in an expense report */
```
NAB. Using "and" hints that we are summing two sets (transactions that are unheld and transactions that are non-reimbursable) where it's actually one set (transactions that are both unheld and non-reimbursable).

Reply:
Make sense, updated!